partner_multi_relation_tabs/models/res_partner.py

```diff
@@ -71,10 +72,11 @@ def add_invisible_extra_field(view, extra_fields, fieldname):
     @api.depends('is_company', 'category_id')
     def _compute_tabs_visibility(self):
         """Compute for all tabs wether they should be visible."""
-        for tab in self._get_tabs():  # get all tabs
-            for this in self:
-                this[tab.get_visible_fieldname()] = \
-                    tab.compute_visibility(this)
+        if "show_relation_tab" in self.env.context:
```

Comment:
I think this change is not really needed (and therefore the change in test also not), because the change in the browse method already prevents this from being called unneeded.

Reply:
oh yes, removed it, thank you